tequilapi/http_routes.go

```diff
@@ -2,16 +2,21 @@ package tequilapi
 
 import (
 	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/ipify"
 	"github.com/mysterium/node/tequilapi/endpoints"
 	"os"
 	"time"
 )
 
+// NewAPIRouter returns new api router with status endpoints
 func NewAPIRouter() *httprouter.Router {
 	router := httprouter.New()
 	router.HandleMethodNotAllowed = true
 
 	router.GET("/healthcheck", endpoints.HealthCheckEndpointFactory(time.Now, os.Getpid).HealthCheck)
 
+	ipifyClient := ipify.NewClient()
+	router.GET("/ip", endpoints.NewCurrentIPEndpoint(ipifyClient.GetPublicIP).CurrentIP)
```

Comment:
Maybe we can make a more generic endpoint and call it /network which can return more network information, not only ip?

Reply:
It's not hard to rename later on if we have to add more fields - for now, all it does is return IP, because that's the user story.